cluster-autoscaler/cloudprovider/gce/mig_info_provider.go

```diff
@@ -151,6 +153,23 @@ func (c *cachingMigInfoProvider) getCachedMigForInstance(instanceRef GceRef) (Mi
 func (c *cachingMigInfoProvider) RegenerateMigInstancesCache() error {
 	c.cache.InvalidateAllMigInstances()
 	c.cache.InvalidateAllInstancesToMig()
+	c.cache.InvalidateMigInstancesState()
+
+	if c.bulkGceMigInstancesListingEnabled {
+		err := c.fillMigInfoCache()
```

Comment:
Can you extract the inside of the if to a separate function?

Reply:
Done